scripts/checkProgress.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+clear
+
+cmd1="sqlite3 data/crypto_trading.db"
```

Comment:
Would be cool if we could invoke this through docker exec so that we can just point it to a container instead (that way we can just hard code the sql file path like we do here as well).
Don't think the current Dockerfile includes sqlite so we'll need to add it there too

Reply:
I guess to keep the best of both worlds we could keep this script as is and just add sqlite to the Dockerfile, the workdir is /app anyway so the relative path here will work fine

Reply:
yes